src/librustdoc/clean/types.rs

```diff
@@ -1054,13 +1054,10 @@ pub(crate) struct CfgInfo {
 impl Default for CfgInfo {
     fn default() -> Self {
         Self {
-            hidden_cfg: [
-                Cfg::Cfg(sym::test, None),
-                Cfg::Cfg(sym::doc, None),
-                Cfg::Cfg(sym::doctest, None),
-            ]
-            .into_iter()
-            .collect(),
+            hidden_cfg: FxHashSet::from_iter(
+                [Cfg::Cfg(sym::test, None), Cfg::Cfg(sym::doc, None), Cfg::Cfg(sym::doctest, None)]
+                    .into_iter(),
```

Comment:
```suggestion
```

(`.into_iter()` not necessary since we're passing things to `from_iter` already)